gui/CMakeLists.txt

```diff
@@ -149,6 +156,10 @@ target_link_libraries(mapgui
   ${QT_LIBRARIES}
 )
 
+if(vtkWebGLExporter_LOADED)
+  target_link_libraries(mapgui vtkWebGLExporter)
```

Comment:
We tried to use `target_compile_definitions` but it's not compatible with `qt4_wrap_cpp` which ignore our definition in its moc generation. So for now, we let `add_definitions` and we are filling a bug report in CMake but not sure they will correct it.